compiler/prelude/TysPrim.hs

```diff
@@ -388,14 +388,11 @@ openBetaTy  = mkTyVarTy openBetaTyVar
 multiplicityTyVar :: TyVar
 multiplicityTyVar = mkTemplateTyVars (repeat multiplicityTy) !! 13  -- selects 'n'
 
--- Create 'count' multiplicity TyVars. Do not give them names that are already
--- present in the 'forbidden' list.
-multiplicityTyVarList :: Int -> [OccName] -> [TyVar]
-multiplicityTyVarList count forbidden = take count $
-                                        filter (not . (`elemOccSet` forbiddenSet) . getOccName) $
```

Comment:
Because multiplicity variables are now metavariables, instead of being inferred. Note that the output of LinearGhci no longer indicates multiplicities.